showcase/app/controllers/components/page-header.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Controller from '@ember/controller';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+import { inject as service } from '@ember/service';
+
+export default class PageHeaderController extends Controller {
+  @service router;
```

Comment:
[question] are you using the router service for anything?

Reply:
My bad, forgot to delete that. Should be updated now.